### PR TITLE
Linting

### PR DESCRIPTION
**Pull request by JakeChampion**
This should be what is needed to make `gulp` pass

### CODE REVIEW

File: client/generator-carousel-admin/js/main.js [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
-var default_url = "https://en.wikipedia.org/wiki/Static_web_page";
+const default_url = 'https://en.wikipedia.org/wiki/Financial_Times';
+const default_duration = 10;
 
-var keyUpTimeout = null,
-	tableBody,
-	templateInputBox;
+let keyUpTimeout;
+let tableBody;
+let templateInputBox;
```
Comment (AdaRoseCannon) [inline client/generator-carousel-admin/js/main.js:7]: :+1:

File: client/common/js/urls.js [hunk 2]
```diff
@@ -1,141 +0,0 @@
-'use strict';
```
Comment (AdaRoseCannon) [inline client/common/js/urls.js:1]: ~~Why do we no longer need this, do we not do url rewrites anymore?~~

Nevermind it was just moved, ideally the move would not be combined with edits to allow git to see that it is just a rename.

File: client/generator-carousel-admin/js/main.js [hunk 3]
```diff
@@ -1,15 +1,14 @@
+/* eslint-env browser */
```
Comment (AdaRoseCannon) [inline client/generator-carousel-admin/js/main.js:1]: This file is not transpiled hence it is still written in es5. If we make these changes we will lose some support.

File: client/common/js/urls.js (hunk 2 above)
Comment (JakeChampion) [inline client/common/js/urls.js:1]: Ah, I didn't know it had to be an individual commit for git to infer it was as rename, thanks for letting me know :)

File: client/generator-carousel-admin/js/main.js (hunk 3 above)
Comment (JakeChampion) [inline client/generator-carousel-admin/js/main.js:1]: After my latest commit, I no longer see any ES6 syntax in the built files :+1: 

```
cat public/build/**/*.js | grep "const "                                                                                              [2.2.3]
      /** @const */EVENTIGNORE = 'ftLabsDelegateIgnore';
```

**Comment by JakeChampion** [comment]
All tests now pass :-)

**Pull request closed without merging**